Drive labels are considered private information and should not be captured in
logs. The ntfs3g logs go to /var/log/messages so ensure the label is not
included in this message.
diff --git a/src/ntfs-3g.c b/src/ntfs-3g.c
index 47b2def..9573816 100644
--- a/src/ntfs-3g.c
+++ b/src/ntfs-3g.c
@@ -3671,8 +3671,7 @@ static void ntfs_close(void)
 		return;
 	
 	if (ctx->mounted) {
-		ntfs_log_info("Unmounting %s (%s)\n", opts.device, 
-			      ctx->vol->vol_name);
+		ntfs_log_info("Unmounting %s\n", opts.device);
 		if (ntfs_fuse_fill_security_context(&security)) {
 			if (ctx->seccache && ctx->seccache->head.p_reads) {
 				ntfs_log_info("Permissions cache : %lu writes, "
@@ -4051,10 +4050,9 @@ static void setup_logging(char *parsed_options)
 	if (strcmp(opts.arg_device,opts.device))
 		ntfs_log_info("Requested device %s canonicalized as %s\n",
 				opts.arg_device,opts.device);
-	ntfs_log_info("Mounted %s (%s, label \"%s\", NTFS %d.%d)\n",
+	ntfs_log_info("Mounted %s (%s, NTFS %d.%d)\n",
 			opts.device, (ctx->ro) ? "Read-Only" : "Read-Write",
-			ctx->vol->vol_name, ctx->vol->major_ver,
-			ctx->vol->minor_ver);
+			ctx->vol->major_ver, ctx->vol->minor_ver);
 	ntfs_log_info("Cmdline options: %s\n", opts.options ? opts.options : "");
 	ntfs_log_info("Mount options: %s\n", parsed_options);
 }
